### PR TITLE
Fix libxdp.so load issue on edge

### PR DESCRIPTION
**Pull request by Himanshu-xilinx**
* Fix libxdp.so loading issue.
* Added stub function.

### CODE REVIEW

**Comment by chvamshi-xilinx** [comment]
This change is required to fix edge platform compilation issues. This build failure got introduced with xbutil2 code changes done for xbutil re-arch. 
Please review. Skeleton functions added to zocl driver now

File: src/runtime_src/core/edge/common/device_edge.cpp [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (C) 2019 Xilinx, Inc
```
Comment (mamin506) [inline src/runtime_src/core/edge/common/device_edge.cpp]: *2020

File: src/runtime_src/core/edge/common/device_edge.h [hunk 2]
```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) 2019 Xilinx, Inc
```
Comment (mamin506) [inline src/runtime_src/core/edge/common/device_edge.h]: *2020

File: src/runtime_src/core/edge/common/system_edge.cpp [hunk 3]
```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2019 Xilinx, Inc
```
Comment (mamin506) [inline src/runtime_src/core/edge/common/system_edge.cpp]: *2020

File: src/runtime_src/core/edge/common/system_edge.h [hunk 4]
```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2019 Xilinx, Inc
```
Comment (mamin506) [inline src/runtime_src/core/edge/common/system_edge.h]: *2020

File: src/runtime_src/core/edge/hw_em/CMakeLists.txt [hunk 5]
```diff
@@ -14,6 +17,12 @@ file(GLOB EM_SRC_FILES
   "${COMMON_EM_SRC_DIR}/*.cpp"
   "${EM_SRC_DIR}/*.h"
   "${EM_SRC_DIR}/*.cpp"
+  "${COMMON_SRC_DIR}/common/*.h"
+  "${COMMON_SRC_DIR}/common/*.cpp"
+  "${COMMON_SRC_DIR}/../common/system.h"
+  "${COMMON_SRC_DIR}/../common/system.cpp"
+  "${COMMON_SRC_DIR}/../common/device.cpp"
+  "${COMMON_SRC_DIR}/../common/device.h"
```
Comment (mamin506) [inline src/runtime_src/core/edge/hw_em/CMakeLists.txt]: Why use file name? Use wildcard is better. Isn't it?
"${COMMON_SRC_DIR}/../common/*.h"
"${COMMON_SRC_DIR}/../common/*.cpp"

File: src/runtime_src/core/edge/user/CMakeLists.txt [hunk 6]
```diff
@@ -12,6 +14,10 @@ file(GLOB XRT_USER_FILES
   "*.h"
   "*.cpp"
   "*.c"
+  "../common/system_edge.cpp"
+  "../common/device_edge.cpp"
+  "../../common/system.cpp"
+  "../../common/device.cpp"
```
Comment (mamin506) [inline src/runtime_src/core/edge/user/CMakeLists.txt]: wildcard?

File: src/runtime_src/core/edge/user/device_linux.cpp [hunk 7]
```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (C) 2019 Xilinx, Inc
```
Comment (mamin506) [inline src/runtime_src/core/edge/user/device_linux.cpp]: All of the new files should be *2020.

File: src/runtime_src/core/edge/hw_em/CMakeLists.txt (hunk 5 above)
Comment (Himanshu-xilinx) [reply src/runtime_src/core/edge/hw_em/CMakeLists.txt]: there are many other files in 'common' folder but we are using specific files only.

Comment (mamin506) [reply src/runtime_src/core/edge/hw_em/CMakeLists.txt]: I'm good with this change.

**Comment by chvamshi-xilinx** [comment]
Merged this change as its blocking edge platform creation

**Pull request merged**